ci/do_ci.sh

```diff
@@ -34,8 +33,8 @@ elif [[ "$1" == "bazel.asan" ]]; then
   cd "${ENVOY_CONSUMER_SRCDIR}"
   echo "Building and testing..."
   # TODO(htuch): This should switch to using clang when available.
-  bazel --batch test ${BAZEL_TEST_OPTIONS} -c dbg --config=asan --test_output=all \
-    --cache_test_results=no @envoy//test/... //:echo2_integration_test
+  bazel --batch test ${BAZEL_TEST_OPTIONS} -c dbg --config=asan @envoy//test/... \
```

Comment:
btw: we should use `@envoy//` prefix across the build, otherwise targets are going to be built twice - once for `//...` and once for `@envoy//...`, since Bazel thinks those are 2 different repositories.

See: https://github.com/bazelbuild/bazel/issues/1248